Documentation/spelling_wordlist.txt

```diff
@@ -107,6 +107,7 @@ Polytechnique
 Portworx
 QCon
 QLogic
+Resynchronization
```

Comment:
Nit: Please remove this entry, it's already covered by the lower-case, hence case-insensitive variant.